tests/conftest.py

```diff
@@ -1674,12 +1674,14 @@ def fixture_context_config(
         },
         "rnafusion": {
             "binary_path": Path("path", "to", "bin", "nextflow").as_posix(),
+            "compute_env": "nf_tower_compute_env",
             "conda_binary": Path("path", "to", "bin", "conda").as_posix(),
             "conda_env": "S_RNAFUSION",
             "launch_directory": Path("path", "to", "launchdir").as_posix(),
             "pipeline_path": Path("pipeline", "path").as_posix(),
             "profile": "myprofile",
             "references": Path("path", "to", "references").as_posix(),
+            "revision": "2.2.0",
```

Comment:
Given our conversation I think that we can break this revision out to a fixture for rnafusion!

Reply:
But since it's part of the cg.yalm config (with the idea of having the rnafusion version locked) I think it makes sense to leave it here

Reply:
resolving